src/cryptominisat.h.in

```diff
@@ -92,8 +92,22 @@ namespace CMSat {
 
         void set_num_threads(unsigned n); //Number of threads to use. Must be set before any vars/clauses are added
         void set_allow_otf_gauss(); //allow on-the-fly gaussian elimination
-        void set_max_time(double max_time); //max time to run to on next solve() call
-        void set_max_confl(int64_t max_confl); //max conflict to run to on next solve() call
+        /**
+         * CPU time (in seconds) that can be consumed before the next call to solve() must return
+         *
+         * Because the elapsed CPU time depends on both the number of
+         * threads, and the activity of these threads, the elapsed time
+         * can wildly differ from wall clock time.
+         * 
+         * Note: Negative values are ignored.
```

Comment:
Please remove this line and just assert-fail. We can put into the docs: "hey, yeah, this blue button next to the red one will cut off the wigs of the aircraft instead of adjust your seat" but most people will be surprised nevertheless :) Assert-fail and just not put it here. If they put a negative value, they'll get an assert fail and they will know it was a bad idea :)

Reply:
This is literally the current behaviour